### PR TITLE
Create an "API" landing page

### PR DESCRIPTION
**Pull request by bucanero**
This PR adds a simple landing page to our `API` section.

### CODE REVIEW

File: docs/api/overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Take a look at the different available APIs to help you build amazing apps on NEAR.
+
+## RPC API
+
+[NEAR RPC API](/docs/api/rpc) provides a simple JSON RPC 2.0 API to interact with the NEAR blockchain.
+
+| API | Description |
+|-----|-------------|
+| [Access Keys](/docs/api/rpc/access-keys) | Retrieve information about access keys. |
```
Comment (thisisjoshford) [inline docs/api/overview.md:15]:
```suggestion
| [Access Keys](/docs/api/rpc/access-keys) | Retrieve information about an account's access keys. |
```

File: docs/api/overview.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+| [Accounts / Contracts](/docs/api/rpc/contracts) | View details about accounts and contracts. |
```
Comment (thisisjoshford) [inline docs/api/overview.md:16]:
```suggestion
| [Accounts / Contracts](/docs/api/rpc/contracts) | View details about accounts and contracts as well as perform contract calls. |
```

File: docs/api/overview.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| [Block / Chunk](/docs/api/rpc/block-chunk) | Query the network and get details about block and chunks. |
```
Comment (thisisjoshford) [inline docs/api/overview.md:17]:
```suggestion
| [Block / Chunk](/docs/api/rpc/block-chunk) | Query the network and get details about specific blocks or chunks. |
```

File: docs/api/overview.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+| [Gas](/docs/api/rpc/gas) | Get gas price for a specific block or hash. |
+| [Protocol](/docs/api/rpc/protocol) | Retrieve current genesis and protocol configuration. |
+| [Network](/docs/api/rpc/network) | Return status information for nodes and validators. |
+| [Transactions](/docs/api/rpc/transactions) | Send and retrieve transactions. |
```
Comment (thisisjoshford) [inline docs/api/overview.md:21]:
```suggestion
| [Transactions](/docs/api/rpc/transactions) | Send transactions and query their status. |
```

**Review by thisisjoshford — CHANGES_REQUESTED** [review]
Left some description suggested edits. Otherwise looks great!

**Comment by bucanero** [comment]
> Left some description suggested edits. Otherwise looks great!

Thanks for the feedback! I've added your suggestions to the PR 🚀

**Pull request merged**